sisl/sparse.py

```diff
@@ -1377,6 +1377,39 @@ def tocsr(self, dim=0, **kwargs):
         return csr_matrix((self._D[idx, dim].copy(), self.col[idx], ptr.astype(int32, copy=False)),
                           shape=shape, **kwargs)
 
+    def transform(self, matrix, dtype=None):
+        """ Apply a linear transformation :math:`R^n \rightarrow R^m` to the :math:`n`-dimensional elements of the sparse matrix.
+
+        Parameters
+        ----------
+        matrix : array_like
+            transformation matrix of shape :math:`m \times n`
+        dtype : `numpy.dtype`, optional
```

Comment:
here one should not have the backtick's then the documentation renders better.

Reply:
fixed